compiler/ccgtypes.nim

```diff
@@ -449,7 +449,8 @@ proc genRecordFieldsAux(m: BModule, n: PNode,
   of nkRecCase:
     if n.sons[0].kind != nkSym: internalError(m.config, n.info, "genRecordFieldsAux")
     add(result, genRecordFieldsAux(m, n.sons[0], accessExpr, rectype, check))
-    let uname = rope(mangle(n.sons[0].sym.name.s) & 'U')
+    let uname = rope(mangle(n.sons[0].sym.name.s) & "_U")
```

Comment:
This line should go away (and a few words explaining why such a prefix has been chosen helps here)

Reply:
Oh, I guess I screwed up picking lines to commit, haha.